README.dev.md

```diff
@@ -192,3 +199,11 @@ python -m twine upload dist/*
 ### (3/3) GitHub
 
 Don't forget to also make a [release on GitHub](https://github.com/ANNUBS/annubes/releases/new). If your repository uses the GitHub-Zenodo integration this will also trigger Zenodo into making a snapshot of your repository and sticking a DOI on it.
+
+## Development conventions
+
+- Branching
+  - When creating a new branch, please use the following convention: `<issue_number>_<description>_<author_name>`.
+  - Always branch from `dev` branch, unless there is the need to fix an undesired status of `main`. See above for more details about the branching workflow adopted.
```

Comment:
after the first release it may make sense to already have it if students start to use the package already. Now the entire dev.md is not properly updated. I'd leave it as it is for now.